azbb-scripts/abbt-cluster.json

```diff
@@ -140,24 +140,34 @@
               },
               "storageAccounts": {
                 "count": 2
-              },
-              "extensions": [              

```

Comment:
The DNS Server (line 59 in this file) is using extensions. Should we be doing alike there as well? Thanks!